build_tools/github_actions/configure_ci.py

```diff
@@ -7,6 +7,16 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """Determines whether CI should run on a given PR.
 
+The following environment variables are required:
+- GITHUB_EVENT_NAME: GitHub event name, e.g. pull_request.
+- GITHUB_OUTPUT: path to write workflow output variables.
+
+When GITHUB_EVENT_NAME is "pull_request", there are additional environment
+variables to be set:
+- PR_TITLE (required): PR title.
+- PR_BODY (optional): PR description.
+- BASE_REF (required): base commit SHA of the PR.
```

Comment:
I don't see PR_TITLE and PR_BODY being used. Are they still necessary?

Reply:
They are still used in `get_trailers()`

Reply:
Ah, right. I missed that these were used before this change. GitHub is very aggressive about collapsing diffs...